lib/db/mysql.js

```diff
@@ -189,7 +190,9 @@ module.exports = function (log, error) {
         tokenId,
         sessionToken.data,
         sessionToken.uid,
-        sessionToken.createdAt
+        sessionToken.createdAt,
+        sessionToken.userAgent,
+        sessionToken.lastAccessTime
```

Comment:
ISTM that `lastAccessTime` should always equal `createdAt` here, maybe we don't need to accept it as an additional parameter.